src/memory-usage.ts

```diff
@@ -80,8 +80,8 @@ class MemoryUsageFeature implements vscodelc.StaticFeature {
 class TreeAdapter implements vscode.TreeDataProvider<InternalTree> {
   private root_?: InternalTree;
 
-  get root(): InternalTree|null { return this.root_; }
-  set root(n: InternalTree|null) {
+  get root(): InternalTree|undefined { return this.root_; }
```

Comment:
again why these null->undefined changes?

Reply:
Addressed in a previous comment, but it's because `private root_?: InternalTree` is already declared as being `InternalTree | undefined`.